antarest/study/service.py

```diff
@@ -911,13 +906,12 @@ def create_study(
             name=study_name,
             workspace=DEFAULT_WORKSPACE_NAME,
             path=str(study_path),
+            author=author,
+            editor=author,
             created_at=now_utc,
             updated_at=now_utc,
             version=f"{version or NEW_DEFAULT_STUDY_VERSION:ddd}",
-            additional_data=StudyAdditionalData(author=author, editor=author),
             directory_id=directory_id,
-            owner=owner,
```

Comment:
Here we don't fill in the owner anymore, this is one of the reasons some tests fail:
for example when a user list studies, it cannot see studies that are not owned by himself, so if the owner is not filled when he creates a study, he will not be able to see it.